decorator.rb

```diff
@@ -0,0 +1,32 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
+end
+
+class Decorator < Nemeable
```

Comment:
- [ ] there is still the same typo here as the previous reviewer asked so please try to fix that